heartbeat/monitor/monitor.go

```diff
@@ -213,6 +223,29 @@ func (monitor *heartbeatV2Monitor) shouldSkipMessage(messageAge time.Duration, p
 	return false
 }
 
+func (monitor *heartbeatV2Monitor) updateMetrics(_ []byte, _ interface{}) {
+	heartbeats := monitor.GetHeartbeats()
+
+	counterActiveValidators := 0
+	counterConnectedNodes := 0
+	for _, heartbeatMessage := range heartbeats {
```

Comment:
this is so sub-optimal, do not have a better idea, this is how the v1 was implemented.

Reply:
Wondering if we can have a dedicated go routine that will call this each 30 seconds or so.

Reply:
indeed this would be lighter..pushed